ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -576,6 +590,9 @@ protected void deleteBlob(final String blobIdStr, final String serviceId, Future
                       e);
                 }
               }
+              if (exception == null && !attemptChunkDeletes) {
+                routerMetrics.skippedGetBlobCount.inc();
```

Comment:
this count may be unreliable because chunks of blobs that are deleted are also triggered with `attemptChunkDeletes == false`

Reply:
Good point, I moved the line to the calling method.